packages/govuk-frontend/src/govuk/components/character-count/character-count.yaml

```
@@ -30,12 +30,12 @@ params:
   - name: label
     type: object
     required: true
-    description: Options for the label component.
+    description: Additional options for the label used by the character count component.
```

I'm not sure about the use of 'additional' here, because to it suggests that this is optional, whereas the character count must have a label of some sort to be accessible.

(It arguably does makes sense in other places… for example in the checkboxes component where we 'promote' the `text` and `html` options, allowing the user to set the label for a checkbox using `items[].text` but provide e.g. additional classes for the label using `items[].label.classes`… API design is hard 😕)

---

Ah we had two variations suggested in https://github.com/alphagov/govuk-design-system/pull/2929#issuecomment-1798183406

* "**Can be used to add a label…**" to set up an optional label
* "**Additional options for the label…**" to configure a mandatory label

i.e. When mandatory, it means "Additional [user] options" on top of our defaults

---

Good point regarding the radios/checkboxes label options, there's a good [Trello card](https://trello.com/c/RxuFrKQS/19-radio-and-checkbox-item-nested-label-components-list-options-that-cant-be-used) for it

---

Fair enough, 'Additional' just sounds like a thing you can ignore to me 🤷🏻‍♂️. But it might just be how I'm interpreting it.

---

You sure? We can drop the word and it looks good still

```suggestion
    description: Options for the label used by the character count component.
```

---

I think this also applies to https://github.com/alphagov/govuk-frontend/pull/4450#discussion_r1397403872 and https://github.com/alphagov/govuk-frontend/pull/4450#discussion_r1397580463 so let's either change all of them to drop 'Additional' or leave it.

Appreciate some thought's gone into it already and it might just be how I'm interpreting it.

---

I've dropped the entire "Additional options for…" to be consistent with other options

Think this is a winner

```yaml
    description: The label used by the character count component.
```